home/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.contrib import admin
+from django.urls import path, include
+from . import views
+
+urlpatterns = [
+    path('', views.index, name='home')
```

Comment:
Yeah we _could_ do this. I felt class-based views deserved a dedicated tutorial/lesson rather than just a blip in a project that otherwise uses all function-based views. I agree with your sentiment - this would be good to add in a refactor video so we can use it as a small intro as I go through the refactor in the final few videos and leave students with some curiosity about CBV :)